draft-ietf-teep-architecture.md

```diff
@@ -787,10 +787,10 @@ the actual content.
 
 # TEEP Broker
 
-A TEE and TAs often do not have the capability to directly communicate
-outside of the hosting device.  For example, GlobalPlatform
-{{GPTEE}} specifies one such architecture.  This calls for a software
+A TEE and TAs do not have the capability to directly communicate
```

Comment:
The removal of the word "often" makes this sentence incorrect.  There are TEEs that can do so (including TrustZone in some cases where you have a trusted peripheral like in a demo we did).

Reply:
Dave, can you explain how you made the TEE communicate directly with the outside world without having to use the REE side? I am not familiar with the details of your demo.